packages/plugin-core/src/test/suite-integ/migration.test.ts

```diff
@@ -338,64 +342,163 @@ suite("Migration", function () {
       });
     });
 
-    test("migrate to 0.70.0 (preview namespace migration)", (done) => {
-      DendronExtension.version = () => "0.70.0";
-      runLegacyMultiWorkspaceTest({
+    describeMultiWS(
+      "GIVEN migration to 0.81.0 (publishing namespace migration)",
```

Comment:
can you also add test cases for migrating from an even earlier version? (eg. 0.6X to 0.8X)

Reply:
since the migration logic covers everything regarding namespace migration (v1, v2, v3, v4 -> v5), I'll add that each of them are working properly given a legacy config. The actual Dendron version will be irrelevant.